hack/jenkins/kicbase_auto_build.sh

```diff
@@ -61,13 +61,17 @@ export KICBASE_IMAGE_REGISTRIES="${GCR_IMG} ${DH_IMG}"
 
 
 # Build a new kicbase image
-yes|make push-kic-base-image
+yes | make push-kic-base-image | tee kic-logs.txt
 
 # Abort with error message if above command failed
 ec=$?
 if [ $ec -gt 0 ]; then
 	if [ "$release" = false ]; then
-		gh pr comment ${ghprbPullId} --body "Hi ${ghprbPullAuthorLoginMention}, building a new kicbase image failed, please try again."
+		err=$(tail -50 kic-logs.txt)
```

Comment:
why not 1000 lines ? (kic logs are small) and we can fit them in a foldable <details> <pre> block

Reply:
I can add the foldable details, my worry is supplying too much useless information. 90% of the time when debugging an issue with these scripts, I just scroll right to the end of the script, and I wanted to avoid having to do this but just showing the end of the script.